packages/router/src/directives/router_link.ts

```diff
@@ -115,19 +115,49 @@ import {UrlTree} from '../url_tree';
 export class RouterLink {
   // TODO(issue/24571): remove '!'.
   @Input() queryParams!: {[k: string]: any};
+  /**
+   * Passed to `Router.createUrlTree` as part of the `NavigationExtras`.
```

Comment:
please use `#` instead of `.` in this case. The community convention is that`#` is used for instance methods and `.` is for static methods.
```suggestion
   * Passed to `Router#createUrlTree` as part of the `NavigationExtras`.
```